packages/react/primitive/src/Primitive.tsx

```diff
@@ -46,10 +46,6 @@ const Primitive = NODES.reduce((primitive, node) => {
     const { asChild, ...primitiveProps } = props;
     const Comp: any = asChild ? Slot : node;
 
-    React.useEffect(() => {
-      (window as any)[Symbol.for('radix-ui')] = true;
```

Comment:
If this is for some reason needed, we can achieve it without `useEffect` as well.